### ISSUE TITLE
Proposal: Drop organizations from the flow until we have a working product

### ISSUE THREAD

**Issue by pachonk** [emerald-rubies/tenacious#130]
I took a step back today and asked myself what could we do to help this project get shipped way faster.
I believe that removing orgs GREATLY decreases complexity and I think that would greatly lower the entry level for this project.
What do you think @Exellin @sanchez2210 @emerald-rubies/emeralds ?

**Comment by sanchez2210** [comment]
It's a possibility. I haven't really wrapped my head around the organizations feature tbh.

**Comment by pachonk** [comment]
Yeah, even I have had trouble thinking about some of the code. Handling 2 of the same routes and trying to force uniqueness on orgs/users this way will be pretty hard and possibly time consuming. Lord knows @Exellin _already_ spent hours fighting this and I really want to lower the entry level of this project so more newer developers can PR more code changes.

Inspiration:
- https://play.google.com/music/m/Dzqeetpqzsiwjepoiv6yds5ms2a?t=The_art_of_over-engineering_your_side_projects-Zeal_Interestings_Podcast
- A talk at South East Ruby Conf discussing "Worse is better" and how shipping fast and worse can actually be better than shipping perfect and slow (using Wordpress as an example, since it's actually pretty toxic inside and some ways to work with it... But it definitely is the leading CMS)
